doc/gpu/USAGE.md

```diff
@@ -222,287 +196,66 @@ and needs explicit deletion.
 
 Example of Pod with generated Resource Claim:
 ```YAML
-apiVersion: resource.k8s.io/v1alpha2
+apiVersion: resource.k8s.io/v1alpha3
 kind: ResourceClaimTemplate
 metadata:
-  name: claim-template-1
-  namespace: default
+  name: claim1
 spec:
   spec:
-    resourceClassName: intel-gpu
+    devices:
+      requests:
+      - name: gpu
+        deviceClassName: gpu.intel.com
 ---
 apiVersion: v1
 kind: Pod
 metadata:
-  name: test-pod-1
+  name: test-inline-claim
 spec:
   restartPolicy: Never
   containers:
   - name: with-resource
     image: registry.k8s.io/e2e-test-images/busybox:1.29-2
-    command: ["sh", "-c", "ls -la /dev/dri/ && sleep 5"]
+    command: ["sh", "-c", "ls -la /dev/dri/ && sleep 60"]
     resources:
       claims:
-      - name: resource1
+      - name: resource
   resourceClaims:
-  - name: resource1
-    source:
-      resourceClaimTemplateName: claim-template-1
+  - name: resource
+    resourceClaimTemplateName: claim1
 ```
 
 #### Customizing resources request
 
-Resource Claim references Resource Class in `resourceClassName` field, and based on it the
-Intel GPU Resource Driver decides what GPUs should be allocated to the Resource Claim. For example,
-shared GPU allocation or exclusive.
-
-ResourceClaim can also reference a GpuClaimParameters in `parametersRef` to customize the GPU
-request. For instance quantity and / or qualities of the GPU.
+ResourceClaim device request can be customized. `count` field specifies how many devices are needed.
+'selectors' is a [CEL](https://github.com/google/cel-spec) filter to narrow down allocation to desired GPUs. For instance, amount of
+memory should be at least 16Gi. The attributes and capacity properties of the GPU can be used in CEL.
```

Comment:
Redundant as there's example of this on the next line:
```suggestion
'selectors' is a [CEL](https://github.com/google/cel-spec) filter to narrow down allocation to desired GPUs.
The attributes and capacity properties of the GPU can be used in CEL.
```

If you really want it, use of "should" in it would less confusing if written e.g. like this:
```suggestion
'selectors' is a [CEL](https://github.com/google/cel-spec) filter to narrow down allocation to desired GPUs. For instance, if the amount of memory should be at least 16Gi, need for that GPU property can be specified with a CEL expression.
```